src/bpmat/BCSRMat.h

```diff
@@ -56,6 +56,9 @@ class BCSRMat : public TACSObject {
   // Free the matrix
   ~BCSRMat();
 
+  // Create a duplicate of the matrix without copying values
+  BCSRMat *createDuplicate();
+
```

Comment:
@gjkennedy using Fortran terminology: a mold() function would allocate but doesn't copy values

Reply:
I think duplicate is a nice description for this. This function will be used infrequently and typically not in instances when you actually want to copy the values of the matrix.